app/scss/partials/_summary.scss

```diff
@@ -168,14 +168,36 @@
 	}
 }
 
+
 // Additional specificity needed to overwrite the position value of g-tooltip
-.Summary__statsButton.Summary__statsButton--absolutely-positioned {
+.Summary__statsButton.Summary__statsButton--absolutely-positioned,
+.Summary__rewardsIcon.Summary__rewardsIcon--absolutely-positioned {
 	position: absolute;
 	right: 0;
-	top: 40px;
-	padding: 6px 3px 3px 9px;
+	padding-top: 6px;
+	padding-bottom: 3px;
 	border: 1px lightgray solid;
+	border-right: none;
 	border-radius: 5px 0 0 5px;
+	background-color: $whisper;
+}
+// Differences in padding improve icon visual alignment
+.Summary__statsButton.Summary__statsButton--absolutely-positioned {
+	top: 40px;
+	padding-right: 3px;
+	padding-left: 9px;
+}
+.Summary__rewardsIcon.Summary__rewardsIcon--absolutely-positioned {
+	top: 85px;
+	padding-right: 6px;
+	padding-left: 10px;
+	text-transform: capitalize;
+}
+// position the star SVG relative to the gift box SVG
+.Summary__rewardsIcon__star {
```

Comment:
again, I'll check this when changes are made and I generate a reward.

Reply:
You can force the value of unreadOffersAvailable in Summary#_renderRewardsIcon to true to test the appearance/layout of the star. That bool is true wwhen the unread_offer_ids array sent from the background is not empty.